crates/pathfinder/src/rpc/v02/types/class.rs

```diff
@@ -0,0 +1,178 @@
+use crate::rpc::serde::U64AsHexStr;
+
+use anyhow::Context;
+use serde::{Deserialize, Serialize};
+use stark_hash::StarkHash;
+
+impl ContractClass {
+    pub fn from_definition_bytes(data: &[u8]) -> anyhow::Result<ContractClass> {
```

Comment:
This takes the place of `Deserialize`, and its required because program requires extra steps to be correct. Since it is just a string, one could still use `serde` directly and then do those decompression and encoding steps, but I think this is more clear and less error-prone.

Reply:
I'm wondering if this could be reworked using `serde_json::value::RawValue`. Not now ofc.